html/partials/patterns.html.haml

```diff
@@ -6,6 +6,11 @@
         :scope => "patterns",
         :title => "Patterns",
         :count => "#{patterns.length} #{Machinery.pluralize(patterns.length, "pattern")}"
+      - if @dpkg_note
```

Comment:
I don't like having a global variable around for that. Could we pass along the description and check for the package_system here directly instead? That way you don't have to touch the `server.rb` anymore, which you shouldn't really for that kind of change.